test/cypress/integration/simplified-account/service-settings/card-payments/updates.cy.js

```diff
@@ -102,34 +112,157 @@ describe('Card payment updates', () => {
       allowApplePay: false
     })
     cy.task('setupStubs', [
-      patchAccountByServiceIdUpdateApplePaySuccess(SERVICE_EXTERNAL_ID, ACCOUNT_TYPE, true)
+      patchAccountByServiceExternalIdAndAccountTypeUpdateApplePaySuccess(SERVICE_EXTERNAL_ID, ACCOUNT_TYPE, true)
     ])
     cy.visit(baseUrl + '/apple-pay')
     cy.get('h1').should('contain.text', 'Apple Pay')
-    cy.get('.service-settings-nav__li--active').within(() => {
-      cy.get('#card-payments').should('contain.text', 'Card payments')
-    })
+    checkSettingsNavigation('Card payments', baseUrl)
     cy.get('input#apple-pay-on').click()
     cy.contains('button', 'Save changes').click()
     cy.get('.govuk-heading-l').should('contain.text', 'Card payments')
   })
 
-  it('should allow update of Google Pay', () => {
+  it('should allow update of Google Pay with active Stripe credential', () => {
     setupStubs({
-      allowGooglePay: false
+      allowGooglePay: false,
+      gatewayAccountPaymentProvider: STRIPE
     })
     cy.task('setupStubs', [
-      patchAccountByServiceIdUpdateGooglePaySuccess(SERVICE_EXTERNAL_ID, ACCOUNT_TYPE, true)
+      patchAccountByServiceExternalIdAndAccountTypeUpdateGooglePaySuccess(SERVICE_EXTERNAL_ID, ACCOUNT_TYPE, true)
     ])
     cy.visit(baseUrl + '/google-pay')
     cy.get('h1').should('contain.text', 'Google Pay')
-    cy.get('.service-settings-nav__li--active').within(() => {
-      cy.get('#card-payments').should('contain.text', 'Card payments')
+    checkSettingsNavigation('Card payments', baseUrl)
+    cy.get('input#google-pay-on').click()
+    cy.contains('button', 'Save changes').click()
+    cy.get('.govuk-heading-l').should('contain.text', 'Card payments')
+  })
+
+  it('should allow update of Google Pay with active Worldpay credential', () => {
+    const googlePayMerchantId = '0123456789abcde'
+    const googlePayMerchantIdError = 'Enter a Google Pay merchant ID'
+    setupStubs({
+      allowGooglePay: true,
```

Comment:
not strictly important but this should be `false` if there's no existing google pay merchant ID

Reply:
sure, there are cases where there could have been a previously set merchant id but the stubs are configured for no existing id so I'll change it